sdk/typespec/typespec_client_core/CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 ### Features Added
 
--   Added `get_async_runtime()` and `set_async_runtime()` to allow customers to replace
-    the default asynchronous runtime with another.
+- Added `get_async_runtime()` and `set_async_runtime()` to allow customers to replace
+the default asynchronous runtime with another.
```

Comment:
Interesting. What changed on your machine the last PR and this one?

Reply:
On whichever machine originally put the spaces in, can you check which formatter claims ownership of markdown files? Would be good to figure out how to prevent flipping back and forth. I'd rather standardize on markdown-lint just because Microsoft Learn does as well, so we'd share formatting with official doc policies.